common/src/main/kotlin/entity/AuditLog.kt

```diff
@@ -138,12 +139,13 @@ public data class AuditLogChange<T>(
                 val oldVal = old?.let { Json.decodeFromJsonElement(key.serializer, old) }
 
                 @Suppress("UNCHECKED_CAST")
-                return AuditLogChange(
+                returnVar = AuditLogChange(
                     new = newVal,
                     old = oldVal,
                     key = key as AuditLogChangeKey<Any?>
                 ) as AuditLogChange<T>
             }
+            return returnVar!!
```

Comment:
So instead of
```kotlin
var r = null
decoder.decodeStructure {
   ...
    r = ...
}
return r!!
```
do
```kotlin
return decoder.decodeStructure {
    ...
    return@decodeStructure ...
}
or
```kotlin
fun deserialize(...) = decodeStructure {
    ...
    lastExpressionReturn()
}
```